protocol/src/main/resources/protocol.xml

```diff
@@ -97,6 +104,7 @@
     <field name="intent" id="6" type="uint8"/>
     <!-- populated when RecordType is COMMAND_REJECTION -->
     <field name="rejectionType" id="7" type="RejectionType"/>
+    <field name="brokerVersion" id="9" type="Version" sinceVersion="2" presence="optional" />
```

Comment:
The main change of this PR. Adding a new field for the broker version. Following https://github.com/real-logic/simple-binary-encoding/wiki/Message-Versioning.